viam_cartographer_test.go

```diff
@@ -325,65 +230,37 @@ func TestNew(t *testing.T) {
 
 		timestamp2, err := svc.GetLatestMapInfo(context.Background())
 		test.That(t, err, test.ShouldBeNil)
+
 		test.That(t, timestamp1.After(_zeroTime), test.ShouldBeTrue)
-		test.That(t, timestamp1, test.ShouldResemble, timestamp2)
+		test.That(t, timestamp2.After(timestamp1), test.ShouldBeTrue)
 
 		test.That(t, svc.Close(context.Background()), test.ShouldBeNil)
 	})
 
-	t.Run("Successful creation of cartographer slam service in non localization mode when feature flag enabled", func(t *testing.T) {
+	t.Run("Fails to create cartographer slam service with no sensor", func(t *testing.T) {
```

Comment:
This merges `Fails to create cartographer slam service with no sensor when feature` and `Fails to create cartographer slam service with no sensor` and `Fails to create cartographer slam service with no sensor when feature flag enabled`